### PR TITLE
Add every two weeks sync meeting notes 2022-01-31

### PR DESCRIPTION
**Pull request by vmx**

### CODE REVIEW

File: meeting-notes/2022/2022-01-31--ipld-sync.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+   - libipld release planned for this week
+
+@rvagg
+ - Lots of playing with bindnode for graphsync, helping find bugs and identify rough areas, @mvdan doing a champion job of making it production-ready
+ - go-car: v2 blockstore can now create just a CARv1 https://github.com/ipld/go-car/pull/288, CLI `car create` now has a `--version` flag so it can make a CARv1.
+ - Big 'ol fix of go-unixfsnode to produce "correct" output (missing Tsize in a few places) and HAMT wasn't working for sharded directories; also came up with some boundary condition tests to match exact go-unixfs bytes
+ - go-ipld-prime dag-cbor length checking on encode to avoid invalid bytes https://github.com/ipld/go-ipld-prime/pull/333
+ - go-ipld-prime dag-cbor EncodedLength() proposal: https://github.com/ipld/go-ipld-prime/pull/346
+ - go-ipld-prime equivalent of https://github.com/rvagg/js-ipld-garbage
+ - Wrote [initial PLN Launchpad IPLD curriculum](https://protocol-labs.gitbook.io/launchpad-curriculum/launchpad-learning-resources/about-2)
```
Comment (rvagg) [inline meeting-notes/2022/2022-01-31--ipld-sync.md:57]:
```suggestion
 - Wrote [initial PLN Launchpad IPLD curriculum](https://protocol-labs.gitbook.io/launchpad-curriculum/launchpad-learning-resources/ipld)
```

this URL got _fixed_ a couple of hours ago so it's probably worth getting it right here

**Review by rvagg — APPROVED** [review]
